### PR TITLE
Task event completion bug fixes

### PR DESCRIPTION
**Pull request by johnbenjamin-hmcts**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-5475
https://tools.hmcts.net/jira/browse/EUI-5489
https://tools.hmcts.net/jira/browse/EUI-5496

### Change description ###
EUI-5475 Task not getting completed when we Multiple tasks
EUI-5489 Validation error message displaying for user after assigning task

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

**Review by waynepHMCTS — APPROVED** [review]

**Review by kuda1992 — APPROVED** [review]
LGTM

File: src/shared/components/case-editor/services/cases.service.ts [hunk 1]
```diff
@@ -274,16 +274,19 @@ export class CasesService {
...
 
   private processTasksOnSuccess(caseData: any, eventData: any): void {
-    // This is used a feature toggle to
-    // control the work allocation
-    if (this.appConfig.getWorkAllocationApiUrl() && !this.isPuiCaseManager()) {
+    // The following code is work allocation 1 related
+    if (this.appConfig.getWorkAllocationApiUrl().toLowerCase() === 'workallocation') {
+      // This is used a feature toggle to
+      // control the work allocation
+      if (this.appConfig.getWorkAllocationApiUrl() && !this.isPuiCaseManager()) {
```
[the remark this answers is not in the document]
Comment (johnbenjamin-hmcts) [reply src/shared/components/case-editor/services/cases.service.ts]: Thank you @kuda1992 I updated the code.

**Review by waynepHMCTS — DISMISSED** [review]

**Review by kuda1992 — DISMISSED** [review]
Looks good

**Review by udaydenduluri33 — APPROVED** [review]
@johnbenjamin-hmcts  thanks for the fix

**Review by udaydenduluri33 — APPROVED** [review]

**Review by udaydenduluri33 — APPROVED** [review]
@johnbenjamin-hmcts  Thanks for the fix

**Review by udaydenduluri33 — DISMISSED** [review]
@johnbenjamin-hmcts thanks for the changes

**Review by udaydenduluri33 — DISMISSED** [review]

**Pull request merged**